src/app/test/service/testRunner.ts

```diff
@@ -11,11 +12,19 @@ export type DetailedTestResult = TestResult & {
   requestDurationMs: number
 }
 
+export async function loadTestCaseModules() {
+  const caseFiles = await glob('../**/*.case.ts', { absolute: true })
+  return caseFiles.map(async (file) => import(file).then((module) => module.default as TestCase))
```

Comment:
We could do `Promise.all` here, and then the `const testCase = await testCaseModule` won't be necessary anymore

Reply:
Done 👍